### ISSUE TITLE
FxLayout and FxLayoutAlign demo

### ISSUE THREAD

**Issue by kek-Sec** [angular/flex-layout#1362]
# Feature Request

#### What is the desired behavior?

I've made the following demo page that I believe could help 

#### What is the use-case or motivation for the desired behavior?
Lack of a live playground/demo

#### Is there anything else we should know?
https://kek-sec.github.io/angular-flex-playground/

**Comment by kek-Sec** [comment]
@CaerusKaru can this be added to the docs&demos project? Should I make a pr?

**Comment by BryanYin** [comment]
Actually, there was one: https://tburleson-layouts-demos.firebaseapp.com/#/docs

**Comment by kek-Sec** [comment]
@BryanYin Yeah I've seen that one this was the core reason I built this because I thought a more dynamic demo was needed, its definately a great reference point tho

**Comment by CaerusKaru** [comment]
@kek-Sec I like this a lot! Do you think you could fold it into the existing demo-app (source code is in this repo)? I'd be happy to review it, get it in, and update the site.

**Comment by kek-Sec** [comment]
@CaerusKaru Thanks for your time! You shall see my pr this weekend!
